how-to/networking/install-dnssec.md

```diff
@@ -0,0 +1,190 @@
+
+After this change, there is no need to restart the service, but it needs to be told to reload its configuration. This can be done with the `rndc` tool:
+
+    sudo rndc reconfig
+
+The server will immediately notice the new configuration and start the process to sign the *example.internal* zone. The journal logs will show the progress, and can be inspected with the command:
+
+    sudo journalctl -u named.service -f
+
+The logs will show something similar to this:
+
+    named[3246]: zone example.internal/IN (unsigned): loaded serial 1
+    named[3246]: zone example.internal/IN (signed): loaded serial 1
+    named[3246]: zone example.internal/IN (signed): receive_secure_serial: unchanged
+    named[3246]: zone example.internal/IN (signed): sending notifies (serial 1)
+    named[3246]: zone example.internal/IN (signed): reconfiguring zone keys
+    named[3246]: keymgr: DNSKEY example.internal/ECDSAP256SHA256/44911 (CSK) created for policy default
+    named[3246]: Fetching example.internal/ECDSAP256SHA256/44911 (CSK) from key repository.
+    named[3246]: DNSKEY example.internal/ECDSAP256SHA256/44911 (CSK) is now published
+    named[3246]: DNSKEY example.internal/ECDSAP256SHA256/44911 (CSK) is now active
+    named[3246]: zone example.internal/IN (signed): next key event: 23-Oct-2024 22:47:12.544
+    named[3246]: any newly configured zones are now loaded
+    named[3246]: running
+    named[3246]: resolver priming query complete: success
+    named[3246]: managed-keys-zone: Key 20326 for zone . is now trusted (acceptance timer complete)
+    named[3246]: zone example.internal/IN (signed): sending notifies (serial 3)
+
+Depending on the zone size, signing all records can take longer.
+
+A few interesting events can be seen in the logs above:
+
+  * Keys were generated for the *example.internal* zone.
+  * The *example.internal* zone became signed.
+  * A *key event* was scheduled. This is bind9 also taking care of the maintenance tasks of the signed zone and its keys.
+  * Since the zone changed, its serial number was incremented (started as 1, now it's 3).
+
+The DNSSEC keys are kept in `/var/cache/bind`:
+
+    -rw-r--r-- 1 bind bind  413 Oct 23 20:50 Kexample.internal.+013+48112.key
+    -rw------- 1 bind bind  215 Oct 23 20:50 Kexample.internal.+013+48112.private
+    -rw-r--r-- 1 bind bind  647 Oct 23 20:50 Kexample.internal.+013+48112.state
+
+This is the bulk of the work. This zone is now signed, and maintained automatically by bind9 using the *default* dnssec policy.
+
+## Verification
+
+The zone that was just signed is almost ready to serve DNSSEC. Let's perform some verification steps.
+
+As it is now, this zone *example.internal* is "disconnected" from the parent zone. Its name was made up for this how-to, but even if it represented a realm domain, it would still be missing the connection to the parent zone. Remember that DNSSEC relies on the chain of trust, and the parent of our zone needs to be able to vouch for it.
```

Comment:
Should we link here to the explanation page?

Reply:
I think it's a bit late in the howto/guide for that, and would look odd to link to the explanation for the first time here, more than halfway through it.